packages/esm-forms-dashboard-app/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@openmrs/esm-forms-dashboard-app",
+  "version": "1.0.0",
+  "license": "MPL-2.0",
+  "description": "Forms dashboard",
```

Comment:
So, dashboard has a specific meaning inside the patient chart, specifically, a configuration of bits on the left menu. I would suggest we rename this `esm-forms-workspace-app` or something similar and change the necessary places. It would also be nice if this description told me anything about hat this does, e.g., "Widget to open forms from the workspace on the patient chart" or something.